packages/tenant-process/src/services/tenantService.ts

```diff
@@ -417,6 +417,60 @@ export function tenantServiceBuilder(
       );
       return updatedTenant;
     },
+    async revokeDeclaredAttribute(
+      {
+        attributeId,
+        organizationId,
+        correlationId,
+      }: {
+        attributeId: AttributeId;
+        organizationId: TenantId;
+        correlationId: string;
+      },
+      logger: Logger
+    ): Promise<Tenant> {
+      logger.info(
+        `Revoking declared attribute ${attributeId} to ${organizationId}`
```

Comment:
We could improve this with `"[...] to organization ${organizationId}"` or `"[...] to tenant ${organizationId}"`